prototypes/ItemRepeaterPrototypes/ItemsRepeaterExperiments/AttachedBehaviors/SelectionBehavior.cs

```diff
@@ -0,0 +1,132 @@
+﻿using Microsoft.UI.Xaml.Controls;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Controls;
+
+namespace ItemsRepeaterExperiments.AttachedBehaviors
+{
+    public class SelectionBehavior : DependencyObject
+    {
+        // Should we store this as an attached property on the repeater?
+        // What would happen with to repeaters and shift click?
+        private static UIElement LastFocused;
+
+        public static bool GetIsEnabled(DependencyObject obj)
+        {
+            return (bool)obj.GetValue(IsEnabledProperty);
+        }
+
+        public static void SetIsEnabled(DependencyObject obj, bool value)
+        {
+            obj.SetValue(IsEnabledProperty, value);
+        }
+
+        public static readonly DependencyProperty IsEnabledProperty =
+            DependencyProperty.RegisterAttached("IsEnabled", typeof(bool), typeof(SelectionBehavior), new PropertyMetadata(false,
+                new PropertyChangedCallback(OnIsEnabledChanged)));
+
+        private static void OnIsEnabledChanged(DependencyObject sender, DependencyPropertyChangedEventArgs args)
+        {
+            if((bool)args.NewValue)
+            {
+                SetIsSelected(sender, GetIsEnabled(sender));
+                (sender as UIElement).PointerPressed += SelectionBehavior_PointerPressed;
+            }
+            else
+            {
+                (sender as UIElement).PointerPressed -= SelectionBehavior_PointerPressed;
```

Comment:
>PointerPressed [](start = 38, length = 14)

I think when IsEnabled is set to false, we can remove the event handler from the sender always, otherwise we might end up adding multiple handlers to the same element if IsEnabled is toggled more than once.

Reply:
I've updated the function to remove the handler everytime the value changes and reregister it if it is being set to true.